btree/btree.c

```diff
@@ -3443,6 +3451,52 @@ static bool index_is_valid(struct level *lev)
 	return (lev->l_idx >= 0) && (lev->l_idx < node_count(lev->l_node));
 }
 
+/**
```

Comment:
shift line 3440 to 3453 after btree_open_tick as these functions are part of btree_get_tick

Reply:
Updated in latest commit.